Directory.Build.props

```diff
@@ -1,15 +1,19 @@
 ﻿<Project>
   <Import Project="$(MSBuildThisFileDirectory)\Dependencies.props" />
   <PropertyGroup>
-    <NetCoreVersion>netcoreapp2.1</NetCoreVersion>
-    <NetStandardVersion>netstandard2.0</NetStandardVersion>
+    <NetCoreVersions>netcoreapp2.1;netcoreapp3.0</NetCoreVersions>
+    <NetStandardVersions>netstandard2.0;netstandard2.1</NetStandardVersions>
     <NetFrameworkVersion>net461</NetFrameworkVersion> <!-- Framework version that should be targeted alongside with .net standard -->
     <LegacyTargetFrameworks>net452;net45</LegacyTargetFrameworks> <!-- Legacy version, required by internal consumers of packages -->
     <TargetFrameworks>$(LegacyTargetFrameworks)</TargetFrameworks>
+    <IsNetStandard>false</IsNetStandard>
+    <IsNetStandard Condition="$(NetStandardVersions.Contains('$(TargetFramework)'))">true</IsNetStandard>
+    <IsNetCore>false</IsNetCore>
+    <IsNetCore Condition="$(NetCoreVersions.Contains('$(TargetFramework)'))">true</IsNetCore>
```

Comment:
Thank you for your contribution. Really nice change, especially with these conditions. 
Could you please move them into a separate property group?

Reply:
Thanks, done.